### DISCUSSION TITLE
Two overlaying search menus ? new interface....

### DISCUSSION THREAD

**Discussion by metamountain** [Comfy-Org/ComfyUI#4733] [General]
with the update i have 2 overlaying search windows. How can i disable that ?

![image](https://github.com/user-attachments/assets/84f7ed0c-4813-43a2-a487-795d6f4a6dfc)

**Comment by JorgeR81** [comment]
You have the old and new search boxes at the same time !

That's a bug. You have to report here: https://github.com/Comfy-Org/ComfyUI_frontend

You should be able to choose one - OR - the other, in the settings. 

![sb](https://github.com/user-attachments/assets/46fc50f3-01e8-417e-b426-fd48c0e63c67)

**Comment by ltdrdata** [comment]
That phenomenon is known to occur due to some custom nodes.
If updating the custom node doesn't resolve the issue, you should use the legacy search box implementation provided in JorgeR81's mention.
